addons/mail/static/src/models/channel.js

```diff
@@ -173,6 +173,12 @@ registerModel({
             compute: '_computeCorrespondentOfDmChat',
             inverse: 'dmChatWithCurrentPartner',
         }),
+        /**
+         * Determines the default display mode of this channel. Should contain
+         * either no value (to display the chat), or 'video_full_screen' to
+         * start a call in full screen.
+         */
+        defaultDisplayMode: attr(),
```

Comment:
**== Out-of-scope of this PR ===**
This field could be renamed so that it's clearer it's related to `CallView`